### DISCUSSION TITLE
How to combine direct children selector (`*:`) with `:not`?

### DISCUSSION THREAD

**Discussion by piotrkulpinski** [tailwindlabs/tailwindcss#12851] [Help]
Hi, 

I'm looking for a way to style direct children elements, but exclude some of them. 
I'm familiar with the children selector `*:bg-black` but not sure how to combine it with `:not` to get what I want.

I tried something like this but it doesn't work:

```html
<ul class="*:[&:not([class*=animate])]:text-black">
  <li class="animate-spin">Loader</li>
  <li>Element</li>
  <li>Element</li>
  <li>Element</li>
</ul>
```

**Comment by makibat** [comment]
From the example you've provided, I assume you want to target the child element by its class, [so here's a Tailwind Play](https://play.tailwindcss.com/1se2xFJKRF).

Hope that helps!
